public/docs/_examples/hierarchical-dependency-injection/dart/lib/hero_card_component.dart

```diff
@@ -0,0 +1,16 @@
+// #docregion
+import 'package:angular2/angular2.dart';
+import 'hero.dart';
```

Comment:
We should use `package:` imports for libs within the same package. (Now that we're recommending not using the `library` directive, I believe it's important not to use relative imports.)